docs/website/blog/2022-06-14-binding-external-service.md

```diff
@@ -0,0 +1,187 @@
+    service.binding/host: path={.metadata.name}.{.metadata.namespace}.svc.cluster.local
+    service.binding: path={.metadata.name}-credentials,objectType=Secret
+spec:
+  type: ClusterIP
+  ports:
+  - port: 6379
+    targetPort: 6379
+```
+
+In this snippet, the first annotation
+
+```
+service.binding/host: path={.metadata.name}.{.metadata.namespace}.svc.cluster.local
+```
+
+indicates to the SBO to inject a `host` variable into the application, with a value computed based on the
+`metadata.name` and `metadata.namespace` of the Service resource. In this example, the value `redis.external-services.svc.cluster.local`
+will be given to the `host` variable.
+
+
+The second annotation
+
+```
+service.binding: path={.metadata.name}-credentials,objectType=Secret
+```
+
+indicates to the SBO to inject the values defined in the Secret, whose name is the name of the Service resource
+followed by `-credentials` (`redis-credentials` in our example), into the application. In this example, the variable `password` 
+with a value `MyEasyPassword` will be injected into the applications's Pod.
+
+## Adding a ServiceBinding to the Devfile
+
+To define a ServiceBinding, we need information (group, version, kind, name and namespace) about the Application and the Service.
+
+In our example, the service is a Kubernetes Service (group "", version "v1" and kind "Service") named `redis`
+in the `external-services` namespace, 
+
+The application will be the Deployment resource (group "apps", version "v1", kind "Deployment") created by odo when you run `odo dev`.
+By convention, the Deployment name will be the name of the Devfile (in the `.metadata.name` field) followed by `-app` (`my-nodejs-app-app` in our example).
+You don't have to specify the namespace, as the Deployment will be in the same namespace as the Deployment.
+
+The option `bindAsFiles` indicates to the SBO to create files into the Pod's container, each file having the name 
+of a credential, and containing the value of the credential.
+
+```
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: binding-to-redis
+spec:
+  application:
+    group: apps
+    version: v1
+    kind: Deployment
+    name: my-nodejs-app-app
+  services:
+  - group: ""
+    version: v1
+    kind: Service
+    name: redis
+    namespace: external-services
+  bindAsFiles: true
+```
+
+You can create a file `kubernetes/redis.yaml` in your directory containing this snippet,
+and add a Kubernetes component into your Devfile referring to this YAML file:
+
+```
+components:
+
+- name: binding-to-redis
+  kubernetes:
+    uri: kubernetes/redis.yaml
+```
+
+By adding this Kubernetes component to your Devfile, when you run `odo dev`, the ServiceBinding resource defined
+in the `kubernetes/redis.yaml` file will be created into the cluster, and the Service Binding Operator will inject
+into the application's Pod the `host` and `password` necessary to connect to the Redis external service.
+
```

Comment:
A suggestion: maybe you can end this article by showing `odo dev` in action, and also how `odo describe binding` can help show more details about the binding. To me, this would give a quick hint at how the injected credentials could be consumed by the application developer.

Reply:
I've added some content to explain how to use the bindings. 
Thanks